src/Cedar/BridgeUnix.c

```diff
@@ -29,12 +29,14 @@
 #include <sys/ioctl.h>
 #include <sys/stat.h>
 
-#ifndef UNIX_OPENBSD
+#if !defined(UNIX_OPENBSD) && !defined(UNIX_SOLARIS)
 #include <net/ethernet.h>
 #endif
 
 #ifdef UNIX_SOLARIS
+#include <stropts.h>
 #include <sys/sockio.h>
+#include <sys/dlpi.h>
```

Comment:
Can we put these in alphabetical order or would that cause issues due to a specific ordering being required?

Reply:
Yes, I thought there was an ordering dependency, but on my test system it seems not to matter.